packages/manager/modules/logs-to-customer/src/components/logTail/LogTail.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useContext } from 'react';
+import { LogsContext } from '../../LogsToCustomer.context';
+
+export default function LogsTail() {
+  const { currentLogKind, logApiBaseUrl } = useContext(LogsContext);
+
+  return (
+    <div className="flex gap-8 flex-col p-8">
+      <h3>Live tail</h3>
+      <ul>
+        <li>
+          CurrentLogKind: <b>{currentLogKind?.displayName}</b>
+        </li>
+        <li>
+          logApiBaseUrl: <b>{logApiBaseUrl}</b>
+        </li>
+      </ul>
```

Comment:
No translation needed ? At least it should be readable

Reply:
this is temporary content, everything in theses components will be reworked.